src/opnsense/mvc/app/controllers/OPNsense/Diagnostics/Api/SystemhealthController.php

```diff
@@ -119,7 +119,7 @@ public function getRRDlistAction()
      * @param int $detail
      * @return array
      */
-    public function getSystemHealthAction($rrd = "", $inverse = 0, $detail = -1)
+    public function getSystemHealthAction($rrd = "", $detail = -1)
```

Comment:
maybe leave "$inverse" in for backward compatibility with a note to remove in 25.7?